cmd/event-received/main.go

```diff
@@ -31,6 +31,12 @@ type uidEvent struct {
 	UID string `json:"uid"`
 }
 
+type changeEvent struct {
```

Comment:
Don't think this is being used anywhere at the moment?

Reply:
ah yeah, left over from when I had the other event